backend/src/run.py

```diff
@@ -7,81 +7,69 @@
 import traceback
 from json import dumps as stringify
 from typing import Any, Dict, List, Optional, TypedDict
+import importlib
+import os
 
 # pylint: disable=unused-import
 import cv2
 from sanic import Sanic
-from sanic.log import logger
+from sanic.log import logger, access_logger
 from sanic.request import Request
 from sanic.response import json
 from sanic_cors import CORS
 
-from nodes.categories import categories, category_order
-
-# pylint: disable=ungrouped-imports,wrong-import-position
-from nodes import image_adj_nodes  # type: ignore
-from nodes import image_chan_nodes  # type: ignore
-from nodes import image_dim_nodes  # type: ignore
-from nodes import image_filter_nodes  # type: ignore
-from nodes import image_iterator_nodes  # type: ignore
-from nodes import image_nodes  # type: ignore
-from nodes import image_util_nodes  # type: ignore
-
-try:
-    import torch
-
-    # Lazily initialize to avoid unnecessary ram usage
-    if torch.cuda.is_available():
-        # pylint: disable=W0212
-        torch.cuda._lazy_init()
-
-    # pylint: disable=unused-import,ungrouped-imports
-    from nodes import pytorch_nodes  # type: ignore
-    from nodes import pytorch_iterators  # type: ignore
-    from nodes import pytorch_alt_nodes  # type: ignore
-except Exception as e:
-    torch = None
-    logger.warning(e)
-    logger.info("PyTorch most likely not installed")
-
-try:
-    import onnx
-    import onnxruntime
-
-    # pylint: disable=unused-import,ungrouped-imports
-    from nodes import onnx_nodes  # type: ignore
-except Exception as e:
-    logger.warning(e)
-    logger.info("ONNX most likely not installed")
-
-
-try:
-    # pylint: disable=unused-import
-    import ncnn_vulkan
-
-    # pylint: disable=unused-import,ungrouped-imports
-    from nodes import ncnn_nodes  # type: ignore
-except Exception as e:
-    logger.warning(e)
-    logger.info("NCNN most likely not installed")
 
-# pylint: disable=unused-import
-from nodes import utility_nodes  # type: ignore
+from nodes.categories import (
+    categories,
+    category_order,
+    PyTorchCategory,
+    ONNXCategory,
+    NCNNCategory,
+)
 from nodes.node_factory import NodeFactory
+from nodes.utils.exec_options import set_execution_options, ExecutionOptions
+
 from base_types import NodeId, InputId, OutputId
 from chain.cache import OutputCache
 from chain.json import parse_json, JsonNode
 from chain.optimize import optimize
 from events import EventQueue, ExecutionErrorData
 from process import Executor, NodeExecutionError, timed_supplier
-from progress import Aborted
+from progress import Aborted  # type: ignore
```

Comment:
Why?

Reply:
My vscode thinks this import is from a separate progress package and I'm tired of seeing this error

Reply:
Uhmm, then are you sure that we aren't? Does it import the right package if you use system python?

Reply:
I assume so, I've never seen it actually have a runtime error, and usually local files take import predenence over global ones

Reply:
For example, I've run into issues before when testing things where I name a file something like `ncnn.py` and then doing `from ncnn import ncnn` results in a circular dependency error since it tries to import itself rather than the actual ncnn package I have installed. So should be fine I think. But maybe it would be safer to just rename our progress package. Unfortunately you can't do relative imports from run.py because python sucks. But anyway I think that would be better resolved in a separate PR anyway.